### DISCUSSION TITLE
Local development does not show a status waiting for verification to users who request signInWithOtp to register.

### DISCUSSION THREAD

**Discussion by fu10ta** [supabase/supabase#26472] [Questions]
In Stg, I confirmed that users who requested signInWithOtp for membership registration are shown a waiting for verification status in Last Sign In column.

However, when I perform the same process on Local, the waiting for verification status is not displayed and the time of the signInWithOtp request is displayed in there.

Please let me know the above idea to make local development smoother.

**Comment by silentworks** [comment]
Can you post screenshots of the what you are mentioning here as I'm having a hard time figuring out what it is. Also when you mention local development are you talking about the Supabase CLI local instance?
